recipe/meta.yaml

```diff
@@ -1,29 +1,33 @@
-{% set version = "5.9.3" %}
+{% set version = "5.10.0" %}
 
 package:
   name: isort
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/i/isort/isort-{{ version }}.tar.gz
-  sha256: 9c2ea1e62d871267b78307fe511c0838ba0da28698c5732d54e2790bf3ba9899
+  sha256: e52ff6d38012b131628cf0f26c51e7bd3a7c81592eefe3ac71411e692f1b9345
 
 build:
   number: 0
+  skip: True  # [py<36]
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
   entry_points:
     - isort = isort.main:main
```

Comment:
Add to entry_points:
`isort-identify-imports = isort.main:identify_imports_main`
see https://github.com/PyCQA/isort/blob/c154ecff496bf9309eaf5d887f4aa4a0abba28a2/pyproject.toml#L102